gum/arch-arm64/gumarm64writer.h

```diff
@@ -51,6 +52,15 @@ struct _GumArm64Writer
   GumMetalArray label_refs;
   GumMetalArray literal_refs;
   const guint32 * earliest_literal_insn;
+
+  GumArm64DataEndian data_endian;
```

Comment:
```suggestion
  gint data_endian;
```

- Let's avoid introducing a new enum here. Our Vala bindings will be using the `ByteOrder` enum from `glib-2.0.vapi`.
- This field should be moved up. The existing three configuration values are in their own section, starting with `target_os`. I'd say this one should go first, as it's more important/fundamental than `target_os`.

Reply:
Done. I kept the `typedef guint GumArm64DataEndian;`, was that right? Also added a comment to it for what the values should be (since the type is otherwise not very constrained).